scripts/cosm/deploy_erc20.js

```diff
@@ -118,7 +118,18 @@ async function main() {
       },
     ],
   };
-  for (const initMsg of [initMsgAsh, initMsgBash]) {
+  const initMsgCash = {
```

Comment:
Ah, here the famous 18 decimals... trying to overflow :wink:

Reply:
Yeah I used Uint64 before but then had to change to to arbitrary length BN